plugins/bundle/plugin_test.go

```diff
@@ -139,13 +139,14 @@ func TestStop(t *testing.T) {
 		t.Fatalf("Error configuring plugin manager: %s", err)
 	}
 
-	baseConf := download.Config{Polling: download.PollingConfig{LongPollingTimeoutSeconds: &longPollTimeout}}
+	triggerPolling := "polling"
+	baseConf := download.Config{Polling: download.PollingConfig{LongPollingTimeoutSeconds: &longPollTimeout}, Trigger: &triggerPolling}
```

Comment:
nit: Use the config constant here instead of redeclaring it?

Reply:
Done